src/app/admin/create-entity/create-entity.component.html

```diff
@@ -367,7 +367,7 @@ <h3>{{usecaseSchema[activeMenuNo]?.title}}</h3>
 
                 <div class="p-3 sm-card"
                     *ngIf="usecaseSchema.length &&  !isShowJson && usecaseSchema[activeMenuNo]?.status != 'PUBLISHED'">
-                    <span class="m-3 p14 mbtn-10 submit-button text-secondary-color" (click)="showAddForm()">
+                    <span class="m-3 p14 btn btn-sec-bg submit-button text-secondary-color" (click)="showAddForm()">
```

Comment:
@suvarnakale If we are not use this mbtn-10 class then can you remove it from .scss file too.

Reply:
yes sure... I will remove the class from .scss file